src/components/dataset/show.vue

```diff
@@ -12,10 +12,7 @@ except according to the terms contained in the LICENSE file.
 
 <template>
   <div id="dataset-show">
-    <page-back v-show="dataExists" :to="[projectPath(), projectPath('entity-lists')]">
-      <template #title>{{ project.dataExists ? project.nameWithArchived : '' }}</template>
-      <template #back>{{ $t('resource.entities') }}</template>
-    </page-back>
+    <breadcrumbs :links="breadcrumbLinks"/>
```

Comment:
add `v-if="dataExists"` so the breadcrumbs (and the rest of the page) wait until they have everything...

dataExists defined by `...resourceStates([project, dataset])`

then in `breadcrumbLinks` below, we'll know that the project data exists.